midend/copyStructures.h

```diff
@@ -54,19 +54,22 @@ namespace P4 {
  */
 class DoCopyStructures : public Transform {
     TypeMap* typeMap;
+    bool ignoreMethods;
```

Comment:
The name of this flag is not entirely suitable. Please add a comment describing what it does.
It skips methods whose output type is a struct; when this flag is false encountering such methods will produce an error message.